solvers/DPGD.py

```diff
@@ -0,0 +1,76 @@
+from benchopt import BaseSolver
+from benchopt.stopping_criterion import SufficientDescentCriterion
+from benchopt import safe_import_context
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from scipy.sparse.linalg import LinearOperator
+    from scipy.sparse.linalg import cg
+    div = import_ctx.import_from('matrice_op', 'div')
+    grad = import_ctx.import_from('matrice_op', 'grad')
+    dual_prox_tv_aniso = import_ctx.import_from('matrice_op',
+                                                'dual_prox_tv_aniso')
+    dual_prox_tv_iso = import_ctx.import_from('matrice_op', 'dual_prox_tv_iso')
+
+
+class Solver(BaseSolver):
+    """Dual Projected gradient descent for analysis formulation."""
+    name = 'Dual PGD analysis'
+
+    stopping_criterion = SufficientDescentCriterion(
+        patience=3, strategy="callback"
+    )
+
+    # any parameter defined here is accessible as a class attribute
+    parameters = {'alpha': [1.],
+                  'use_acceleration': [False, True]}
```

Comment:
This is not used anywhere?

Reply:
I haven't added codes about acceleration, because without acceleration the solvers can't converge yet.